packages/server/modules/core/graph/schemas/streams.graphql

```diff
@@ -32,13 +32,24 @@ type Stream {
   updatedAt: DateTime!
   collaborators: [StreamCollaborator]!
   size: String
+  """
+  Date when you favorited this stream. `null` if stream isn't viewed from a specific user's perspective or if it isn't favorited.
+  """
+  favoritedDate: DateTime
+  # How many times this stream has been favorited
+  favoritesCount: Int!
 }
 
 extend type User {
   """
   All the streams that a user has access to.
   """
   streams(limit: Int! = 25, cursor: String): StreamCollection
+
+  """
+  All the streams that a user has favorited
+  """
+  favoriteStreams(limit: Int! = 25, cursor: String): StreamCollection
```

Comment:
Makes more sense for this to be a User field not a top level query.

(It's kinda weird how you can do `User.streams` and also just `streams` and both queries nearly do the same thing.)